### DISCUSSION TITLE
Support for OpenJDK

### DISCUSSION THREAD

**Discussion by jjkcharles** [microsoft/openjdk#28] [Q&A] [answered]
The page for OpenJDK says this "The Microsoft Build of OpenJDK is a new no-cost **long-term supported distribution**". Can you please elaborate on the support piece of it - Is it only the Azure customers who would be offered support OR any one who installs it on-prem or even on their local machine would be eligible for support?

**Comment by RyanHir** [comment, marked as answer]
I would assume the LTS means that it is supported as long as upstream supports it, not for a specific use case. For example, Oracle's (and OpenJDK's) Java 11 is a LTS release, and they are not tied to a specific platform or use-case to the best of my knowledge.

**Reply by ghost** [reply]
LTS means that they're going to take the responsibility of incorporating security fixes for some period of time _and release new binaries_ containing those patches.  A vendor offering LTS may be required to backport patches from later OpenJDK versions, too.  Any vendor can make any OpenJDK version a LTS release, but most are following Oracle's cadence to-date.

Note that the OpenJDK Project itself does _not_ offer LTS release binaries.  You have to get those from Oracle (paid license) or another vendor like Azul or Microsoft.  The "stewardship" of LTS branches of the upstream is sponsored by a main contributor/member of the OpenJDK Project, and Java 8 and Java 11 are [stewarded by Red Hat](https://www.theserverside.com/news/252461945/Red-Hat-replaces-Oracle-as-OpenJDK-8-OpenJDK-11-steward) last I heard.

**Reply by karianna** [reply]
We will also clarify our Azure Commercial Support (that is, you can raise a ticket) when we announce our General Availability (GA).
